service.iptv.manager/addon.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="service.iptv.manager" name="IPTV Manager" version="0.2.1+matrix.1" provider-name="Michaël Arnauts">
+<addon id="service.iptv.manager" name="IPTV Manager" version="0.2.2+matrix.1" provider-name="Michaël Arnauts">
     <requires>
         <import addon="xbmc.python" version="3.0.0"/>
         <import addon="script.module.dateutil" version="2.6.0"/>
-        <import addon="pvr.iptvsimple" version="3.9.8"/>
+        <import addon="pvr.iptvsimple" version="3.8.8"/>
```

Comment:
why is this change? Is it because of the ffmpegdirect known bugs?
cc @phunkyfish

Reply:
It's because some users have been reporting that on debian buster, an `apt install kodi-pvr-iptvsimple` will install `3.8.8-1~buster`, and since I don't require on anything special from 3.9.8, I can just lower the minimum requirements.

Reply:
sounds reasonable